src/main/java/org/symphonyoss/symphony/messageml/elements/ListItem.java

```diff
@@ -31,6 +34,12 @@ public ListItem(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  @Override
+  void validate() throws InvalidInputException {
+    super.validate();
+    assertParentAtAnyLevel(Arrays.asList(OrderedList.class, BulletList.class));
```

Comment:
Are you sure about this? Doesn't "li" has to be a direct child of ol/li ?

Reply:
I was thinking about, for instance, a possible <div> between them.  Could this happen? WDYT?

Reply:
> I was thinking about, for instance, a possible div element between them. Could this happen? WDYT?

From the HTML specification, "ul" and "ol" can only have "li"as their child. In fact, I think the validation for the parent element is checking this even (see assertContentModel called on the "ul" and "ol" spec). I would adjust this to be asserting the direct parent